azkaban_executor_sg.tf

```diff
@@ -28,3 +28,33 @@ resource "aws_security_group_rule" "allow_azkaban_webserver_ingress_azkaban_exec
   security_group_id        = aws_security_group.azkaban_executor.id
   source_security_group_id = aws_security_group.azkaban_webserver.id
 }
+
+resource "aws_security_group_rule" "allow_azkaban_executor_egress_azkaban_webserver" {
+  description              = "Allows azkaban webserver to access azkaban executor"
+  type                     = "egress"
+  to_port                  = jsondecode(data.aws_secretsmanager_secret_version.workflow_manager.secret_binary).ports.azkaban_webserver_port
+  protocol                 = "tcp"
+  from_port                = jsondecode(data.aws_secretsmanager_secret_version.workflow_manager.secret_binary).ports.azkaban_webserver_port
+  security_group_id        = aws_security_group.azkaban_executor.id
+  source_security_group_id = aws_security_group.azkaban_webserver.id
+}
+
+resource "aws_security_group_rule" "azkaban_executor_egress_internet_proxy" {
+  description              = "Allow Azkaban executor internet access via the proxy"
+  type                     = "egress"
```

Comment:
Isn't this ingress since its traffic from the internet to our network, and similiary the one below being egress since the source is our network to the external network (internet)?

Reply:
`security_group_id`  is the security group it is applied to `source_security_group_id` although confusing is the destination security group